src/clients/inst_kickoff.rb

```diff
@@ -474,17 +429,9 @@ def createmdadm
       end
 
       # get the current raid configuration
-      out = Convert.to_map(
-        SCR.Execute(
-          path(".target.bash_output"),
-          Ops.add(
-            Ops.add(
-              Ops.add("chroot '", String.Quote(Installation.destdir)),
-              "' "
-            ),
-            "mdadm -Ds"
-          )
-        )
+      out = SCR.Execute(
+        path(".target.bash_output"),
+        "/usr/bin/chroot #{Installation.destdir.shellescape} mdadm -Ds"
```

Comment:
Do we still need this? Maybe it's already handled by the `libstorage-ng`... Again, this code is ~13 years old... :open_mouth:

Reply:
I do not know. @ancorgs ^^^